app/celery/provider_tasks.py

```diff
@@ -101,8 +101,6 @@ def send_sms_to_provider(self, service_id, notification_id):
         current_app.logger.info(
             "SMS {} sent to provider at {}".format(notification_id, notification.sent_at)
         )
-        statsd_client.incr("notifications.tasks.send-sms-to-provider")
-        statsd_client.timing("notifications.tasks.send-sms-to-provider.task-time", monotonic() - task_start)
```

Comment:
does statsd have the concept of migrating/renaming/simlinking the old spelling to the new one?